packages/cardano-services/src/StakePool/HttpStakePoolMetadata/HttpStakePoolMetadataService.ts

```diff
@@ -66,6 +66,8 @@ export const createHttpStakePoolMetadataService = (
       throw error;
     }
   },
+
+  // eslint-disable-next-line max-statements
```

Comment:
This indicates that function should be refactored: simply extracting some functions would make it cleaner.

Reply:
0fda0431b should solve this